### DISCUSSION TITLE
检查中奖不会检查@信息导致错过

### DISCUSSION THREAD

**Discussion by 1974988280** [shanmiteko/LotteryAutoScript#322] [General]
一般中奖后，up不发私信，只会发@信息，导致无法检测中奖错过奖品
![QQ截图20230921141745](https://github.com/shanmiteko/LotteryAutoScript/assets/82140966/506ba179-15cc-491a-a913-164c4859a518)
![QQ截图20230921141824](https://github.com/shanmiteko/LotteryAutoScript/assets/82140966/837d8fb0-f8f2-44f3-a471-3672a600c12a)
![QQ截图20230921141852](https://github.com/shanmiteko/LotteryAutoScript/assets/82140966/7c5f660c-d795-4542-95dd-0315be469312)

**Comment by shanmiteko** [comment]
会检查啊

**Reply by 1974988280** [reply]
这是9月19的日志，都没有监测到
![QQ截图20230921170844](https://github.com/shanmiteko/LotteryAutoScript/assets/82140966/765a3041-d946-40ba-b735-ca8f6c01a52f)
![QQ截图20230921170825](https://github.com/shanmiteko/LotteryAutoScript/assets/82140966/73393c8c-2707-435f-b1d5-d0d00f2d2b3d)

**Reply by 1974988280** [reply]
这是20号的
![20号](https://github.com/shanmiteko/LotteryAutoScript/assets/82140966/77e469a4-959c-4b08-9827-715472afc926)
